tor/strings/en_US.yml

```diff
@@ -56,6 +56,8 @@ responses:
       Since this is your first one, please make sure to message the mods if you have any questions about the process (and / or join our [Discord](https://discord.gg/ZDDE3Pj) to talk to other volunteers). We're here to help!
 
       Please respond with `done` when complete so we can check this one off the list!
+      
+      We have a complete tutorial that shows how to transcribe step-by-step and how to fix the most common issues. We suggest you to look at it [here](https://www.reddit.com/r/TranscribersOfReddit/wiki/tutorial)!
```

Comment:
I love that the YAML formatting of the messages makes things more accessible to making changes. 😁